src/libm/sleefsimddp.c

```diff
@@ -955,7 +955,7 @@ EXPORT CONST VECTOR_CC vdouble xcos_u1(vdouble d) {
 #define XMODF xmodf
 #endif
 
-TYPE2_FUNCATR vdouble2 XSINCOS(vdouble d) {
+TYPE2_FUNCATR VECTOR_CC vdouble2 XSINCOS(vdouble d) {
```

Comment:
@yuyichao Please make sure that the declaration of `sincos` with this signature is also decorated with the `VECTOR_CC` in `sleef.h`.

Reply:
I caught this because I get a linking error when the user code expected the calling convention from sleef.h but the definition code does not provide it so I believe it's already decorated in `sleef.h`. My understanding of `mkrename.c` also suggest that this should be the case.

Reply:
Yep, you are right, I verified on master, the VECTOR_CC is specified  for all versions of `sincos`:

```
frapet01@hw-a21-8:~/projects/sleef/build (master)$ cat include/sleef.h | grep sincos | grep aarch64
IMPORT CONST Sleef_float64x2_t_2 Sleef_sincosd2_u35advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_finz_sincosd2_u35advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_sincosd2_u10advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_finz_sincosd2_u10advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_sincospid2_u05advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_finz_sincospid2_u05advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_sincospid2_u35advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float64x2_t_2 Sleef_finz_sincospid2_u35advsimd(float64x2_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_sincosf4_u35advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_finz_sincosf4_u35advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_sincosf4_u10advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_finz_sincosf4_u10advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_sincospif4_u05advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_finz_sincospif4_u05advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_sincospif4_u35advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
IMPORT CONST Sleef_float32x4_t_2 Sleef_finz_sincospif4_u35advsimd(float32x4_t) __attribute__((aarch64_vector_pcs));
```